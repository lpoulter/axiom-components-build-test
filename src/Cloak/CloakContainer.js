import React, { Component } from 'react';
import Base from '../Base/Base';

export default class CloakContainer extends Component {
  render() {
    return (
      <Base cloakContainer { ...this.props } />
    );
  }
}
