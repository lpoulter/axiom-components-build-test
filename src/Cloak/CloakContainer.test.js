import React from 'react';
import renderer from 'react-test-renderer';
import CloakContainer from './CloakContainer';

const getComponent = (props = {}) =>
  renderer.create(
    <CloakContainer { ...props }>
      Lorem Ipsum
    </CloakContainer>
  );

describe('CloakContainer', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
