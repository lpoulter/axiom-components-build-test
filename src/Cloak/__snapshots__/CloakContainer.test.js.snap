// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloakContainer renders with defaultProps 1`] = `
<div
  className="ax-cloak__container"
>
  Lorem Ipsum
</div>
`;
